benchmark-framework/src/main/java/io/openmessaging/benchmark/Workload.java

```diff
@@ -28,8 +28,12 @@
     /** Number of partitions each topic will contain */
     public int partitionsPerTopic;
 
+    public String keyDistributor;
```

Comment:
Could we just use the enum type here? Jackson should be able to match it from a String on the yaml file.

Reply:
Yes sure, but I'll need to verify that it would proper perform in cases when value is not set and if value is not in ENUM